src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -595,7 +595,7 @@ public abstract partial class JsonNode
         public System.Text.Json.Nodes.JsonObject AsObject() { throw null; }
         public System.Text.Json.Nodes.JsonValue AsValue() { throw null; }
         public string GetPath() { throw null; }
-        public virtual TValue GetValue<TValue>() { throw null; }
+        public virtual T GetValue<T>() { throw null; }
```

Comment:
cc: @terrajobst, @bartonjs

Reply:
Seems fine to me?

Reply:
Same here. I don't think this needs an API review.

Reply:
Thanks